### PR TITLE
Storybook v5 and Storycap

### PR DESCRIPTION
**Pull request by hassoncs**
Upgrade Storybook from v4 to v5
Upgrade our screenshot setup from `storybook-chrome-screenshot` to `storycap`

Most of the changes are just renaming. The main changes are in the `config.js` for storybook and `package.json` that starts up `storycap`.

🔥🔥🔥 This speeds up the build by x3! From ~1h build times to 20m!
image

### CODE REVIEW

File: packages/webviz-core/src/panels/Plot/index.stories.js [hunk 1]
```diff
@@ -151,7 +151,6 @@ const fixture = {
   activeData: {
     startTime: { sec: 0, nsec: 202050 },
     endTime: { sec: 24, nsec: 999997069 },
-    currentTime: { sec: 0, nsec: 750000000 },
```
Comment (janpaul123) [inline packages/webviz-core/src/panels/Plot/index.stories.js:154]: Why this change?

File: stories/config.js [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
-addDecorator(initScreenshot());
-setScreenshotOptions({
-  // if this function is present on the page, we wait for the promise it returns to resolve before taking a screenshot.
-  waitFor: "waitFor",
-  delay: 100, // Small delay for rerenders that some components do.
-  viewport: {
-    width: 1001,
-    height: 745,
+addDecorator((storyFn) => {
+  // TODO: [EP-26051] This can be removed when modals remove themselves
```
Comment (janpaul123) [inline stories/config.js:28]: We usually avoid internal JIRA tags in the open source repo.

Comment (hassoncs) [reply stories/config.js:28]: Ah, good catch. I'll remove it here but leave it in the closed source repo 👍

File: packages/webviz-core/src/panels/Plot/index.stories.js (hunk 1 above)
Comment (hassoncs) [reply packages/webviz-core/src/panels/Plot/index.stories.js:154]: I think this accidentally slipped in, will revert

**Comment by hassoncs** [comment]
@vidaaudrey, sadly there's no naming pattern I could use that gets the before/after image paths to match :( Storycap has a `--flat` flag, but it still returns different paths
